### ISSUE TITLE
Button disapeared/turns white when closing style editor

### ISSUE THREAD

**Issue by excitedbox** [kaiakz/walv#13]
When selecting a button, opening the style editor and then closing it the button turns white.

Recreate problem:
-Place button
-Place object
-Select button
-Open style editor
-Close style editor

**Comment by kaiakz** [comment]
The style editor only supports `text`, maybe walv applys the wrong style for the button(see `wrapper.js`), `code.push(`${id}.set_style(lv.label.STYLE.MAIN,s)`)`

**Comment by user446** [comment]
Actually happens with any widget added on a screen.
They just become invisible.

**Comment by excitedbox** [comment]
Yes it seems to be that the ID is not passed on to the style editor.
